HandleNHDeleteInstallationCallOrchestratorLegacy/index.ts

```diff
@@ -0,0 +1,10 @@
+﻿import * as df from "durable-functions";
+
+import { handler } from "./handler";
+
+const orchestrator = df.orchestrator(handler);
+
+export const orchestratorName =
+  "HandleNHDeleteInstallationCallOrchestratorLegacy";
```

Comment:
There is a hard bind between this variable and the folder in which this file is contained (hence the actual name of the orchestrator).
Despite of that, there's no mechanism that ensures that this constraint is satisfied. I'd rather not introduce this variable and delegare the caller to use the correct string.

Anyway, I see the point and I agree. We planned to build a tool that could help us enforce type safety when using durable functions, see here: https://www.pivotaltracker.com/story/show/174859888